src/EA.Weee.RequestHandlers.Tests.Unit/ExampleXML/v3-same-producer-name.xml

```diff
@@ -2,7 +2,7 @@
 <!-- edited with XMLSpy v2009 sp1 (http://www.altova.com) by Environment Agency (Capgemini UK plc) -->
 <!--Sample XML file generated by XMLSpy v2009 sp1 (http://www.altova.com)-->
 <scheme xmlns="http://www.environment-agency.gov.uk/WEEE/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.environment-agency.gov.uk/WEEE/XMLSchema G:\NOTSFI~1\PRRS\PROGRA~1\PRSD\2014\DISCOVERY\WEEE(S~1\TSU\TESTXS~1\XSDSCH~2.XSD">
-	<XSDVersion>3.07</XSDVersion>
+	<XSDVersion>3.08</XSDVersion>
```

Comment:
there are various XML files in this directory you've not updated; should you have done? E.g. `v3-badly-damaged.xml`, `v3-noproducers.xml`, `v3-slightly-invalid.xml`.

Reply:
the point is that if we restrict the version to 3.08, then these invalid files will be rejected but it might be not for the correct reason, so we would no longer be testing the validation the tests intended. It depends on the error returned.